src/utils/bitmap_from_file.cpp

```diff
@@ -41,7 +41,7 @@
 #include <png.h>
 
 #include <cerrno>
-
+#include <iostream>
```

Comment:
?

Reply:
I had included for debug with std::cout directly but I didn't forget it for remove later ^^